### ISSUE TITLE
Cannot import/export regions when image has invalid or non-existent WCS headers

### ISSUE THREAD

**Issue by veggiesaurus** [CARTAvis/carta-backend#334]
Some images don't have WCS information. For example, the "LMCq.fits" file has a minimal header, shown below. As such, it has no WCS information. The backend should still be able to export regions defined for this file in _pixel_ coordinates (although it's totally fine for it to fail to export in WCS). Currently, the backend fails to export regions, and displays `Export carta region type 3 failed: AnnotationBase: Coordinate system has no direction coordinate`. 

LMCq.fits header:
```
NAXIS1 = 10482
CRVAL1 = 85.0539
CRPIX1 = 5241
CRVAL2 = -70.7087
NAXIS2 = 10448
NAXIS = 2
CRPIX2 = 5224
DATE-OBS = 2017-10-21T01:29:00
```

**Comment by pford** [comment]
Merged fixes for seg fault in release 1.2 branch for patch.  Now sends "no direction coordinate" error message for wcs export.  Can export DS9 pixel coordinates, tested with region import in SAO DS9 viewer.

TBD: merge these changes into dev and fix pixel CRTF import/export and DS9 import.  Both use imageanalysis Annotation regions, which currently require a direction coordinate.

**Comment by Kechil** [comment]
Document for now.
